app/src/main/java/org/mifos/mobile/data/DataManagerTransaction.kt

```diff
@@ -0,0 +1,14 @@
+package org.mifos.mobile.data
+
+import io.reactivex.Observable
+import org.mifos.mobile.models.entity.Transaction
+import org.mifos.mobile.models.entity.TransactionInfo
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class DataManagerTransaction @Inject constructor(private val paymentHubApiManager: PaymentHubApiManager) {
```

Comment:
- add it under `api` package, rename it to `PaymentHubDataManager` or something better
- remove `data` package, I don't think it needed